starter_code/queries.md

```diff
@@ -51,33 +57,37 @@
 <!-- Your Code Goes Here -->
 
 ### 12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
+filter: {"acquisition.price_amount":{$gte:10000000},founded_year:{$lt:2000}}
```

Comment:
{"acquisition.price_amount":{$gt:10000000},founded_year:{$lt:2000}}

just greater than (not equal)